docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -0,0 +1,452 @@
+
+- Create the Meter once, store it in a static variable or DI container, and use that instance as long as needed. Each library or library subcomponent can
+(and often should) create its own <xref:System.Diagnostics.Metrics.Meter>. Consider creating a new Meter rather than reusing an existing one if you anticipate
+app developers would appreciate being able to enable and disable the groups of metrics separately.
+
+- The name passed to the <xref:System.Diagnostics.Metrics.Meter> constructor has to be unique to avoid conflicts with any other Meters. Use a dotted hierarichal
+name that contains the assembly name and optionally a subcomponent name. If an assembly is adding instrumentation for code in a second, independent assembly, the name
+should be based on the assembly that defines the Meter, not the assembly whose code is being instrumented.
+
+- The <xref:System.Diagnostics.Metrics.Meter> constructor version parameter is optional. We recommend that you provide a version in case you release multiple versions
+of the library and make changes to the instruments.
+
+- .NET doesn't enforce any naming scheme for metrics, but by convention all the .NET runtime libraries have metric names using '-' if a separator is needed. Other metric
+ecosystems have encouraged using '.' or '_' as the separator. Microsoft's suggestion is to use '-' in code and let the metric consumer such as OpenTelemetry or
+Prometheus convert to an alternate separator if needed.
+
+#### Viewing the new metric
+
+There are many options to store and view metrics. This tutorial uses the [dotnet-counters](./dotnet-counters.md) tool which is useful for ad-hoc analysis. You can also see
+the [metrics collection tutorial](metrics-collection.md) for other alternatives. If the [dotnet-counters](./dotnet-counters.md) tool is not already installed use the SDK
+to install it:
+
+```dotnetcli
+> dotnet tool update -g dotnet-counters
+You can invoke the tool using the following command: dotnet-counters
+Tool 'dotnet-counters' (version '5.0.251802') was successfully installed.
+```
+
+While the sample app above is still running, list the running processes in a 2nd shell to determine the process ID:
+
+```dotnetcli
+> dotnet-counters ps
+     10180 dotnet     C:\Program Files\dotnet\dotnet.exe
+     19964 metric-instr E:\temp\metric-instr\bin\Debug\netcoreapp3.1\metric-instr.exe
+```
+
+Find the ID for the process name that matches the sample app and have dotnet-counters monitor the new counter:
+
+```dotnetcli
+> dotnet-counters monitor -p 19964 HatCo.HatStore
+Press p to pause, r to resume, q to quit.
+    Status: Running
+
+[HatCo.HatStore]
+    hats-sold (Count / 1 sec)                          4
+```
+
+As expected we can see that HatCo store is steadily selling 4 hats each second. 
+
+### Types of Instruments
+
+Above the example created a <xref:System.Diagnostics.Metrics.Counter> instrument but there are several different types available. Instruments differ in two ways:
```

Comment:
???

```suggestion
The above example created a <xref:System.Diagnostics.Metrics.Counter> instrument but there are several different types available. Instruments differ in two ways:
```

Reply:
Your suggestion did capture my intended meaning. Not sure if your "???" meant there was something else you found confusing as well?

Reply:
I found it hard to understand (perhaps due to my bad English). I interpreted it as "In the example we've only demonstrated the <xref:System.Diagnostics.Metrics.Counter> instrument, but there are more instrument types available."

Reply:
Your English is pretty sharp if you ask me ; ) Your phrasing was clearer so I adopted it.